### PR TITLE
Remove repeated content

### LINKED ISSUE #47: Remove repeated parts from the remaining chapters (the original book chapters)
Description:
--------------
After adding the training notebooks 01 to 04 at the beginning of the CUQI-Book, we need to identify and remove parts of the original chapters that were used for the course that are repeated in the training material (01 to 04) and remove them.

Definition of done:
--------------------
- [ ] Repeated parts are identified 
- [ ] Removing the repeated parts is implemented, reviewed and merged

### PR DESCRIPTION
**Pull request by amal-ghamdi**
closes #47

### CODE REVIEW

**Review by jakobsj — CHANGES_REQUESTED** [review]
Really nice reorganization and simplification. The book is really starting to look excellent!

Some minor suggestions (not sure if I'm viewing the completely latest version?):

- [x] There is still a "remarks" section that reads as if belong to the June course here https://amal-ghamdi.github.io/CUQI-Book-test/chapter01/introduction_overview_of_cuqipy.html
- [x] Put our names as authors on the front page
- [x] What is called chapter 3 in the menu has a filename called chapter A, probably good to rename as chapter 3 for simplicity: https://amal-ghamdi.github.io/CUQI-Book-test/chapter0A/chapter0A.html
- [ ] Chapter 1 startpage has chapter learning objectives (https://amal-ghamdi.github.io/CUQI-Book-test/chapter01/chapter01.html), suggest to add a sentence or two before the learning objective explaining what the chapter is about. 
- [ ] Then other chapters have empty startpages eg chapter 2 would benefit from also having a short intro and chapter learning objectives added: https://amal-ghamdi.github.io/CUQI-Book-test/chapter02/chapter02.html
- [ ] The five little stories are really great as it nicely conveys insights about different samplers. I wonder if it would be beneficial to organize as five separate, shorter files, to keep more manageable and allow navigating them in menu?

**Comment by amal-ghamdi** [comment]
Many thanks @jakobsj for your review. I addressed your comments. I implemented the first 3 and copied the last three in another issue #57 (#56). 

I also added some instructions on how to run the book interactively, but this will be possibly revisited as part of issue #53

The test book reflects the changes in this PR:  https://amal-ghamdi.github.io/CUQI-Book-test/

- [ ] One last note, I am not sure about author's order, so we can discuss that

**Comment by jakobsj** [comment]
@amal-ghamdi is there a rendered version I could look at?

**Comment by amal-ghamdi** [comment]
> https://amal-ghamdi.github.io/CUQI-Book-test/

@jakobsj yes I pushed the same branch to the test book and merged it to the test book main https://amal-ghamdi.github.io/CUQI-Book-test/ (this should be identical to what will this PR produce)

**Review by jakobsj — APPROVED** [review]
Thanks for addressing the comments. Happy with the changes and good to see the remaining ones transferred.

File: chapter01/introduction_overview_of_cuqipy.md [hunk 1]
```diff
@@ -101,29 +101,13 @@ See setting up a Bayesian model in CUQIpy in 4 steps [here](https://cuqi-dtu.git
...
-- Disclaimer: keep in mind that notation may vary among different course components.
-- Throughout we will introduce many tools in CUQIpy and discuss 
-    - how to use them
-    - for what purpose
-    - and identifying the right tool
-- Throughout all the course components, including CUQIpy, you will end up with more tools in your BIP toolbox!
-![toolbox](../images/toolbox3.jpeg)
-
 
 ### How to run the code and solve the coding exercises in this mini-book? <a class="anchor" id="running-the-code"></a>
```
Comment (chaozg) [inline chapter01/introduction_overview_of_cuqipy.md:105]: Just a thought: this is a vital piece of information for using this book, and it's a pit that it is somehow hidden in a section called "Overview of CUQIpy". Do you think it deserves a more noticeable place, for example somewhere before chapter01?

File: chapter07/gradient_based_sampling_with_cuqipy.ipynb
Comment (chaozg) [inline chapter07/gradient_based_sampling_with_cuqipy.ipynb:1]: The file name `gradient_based_sampling_with_cuqipy.ipynb` is a bit misleading (to ourselves when editing) as this notebook contains more "stories" than just gradient sampling. What do you think?

**Review by chaozg — APPROVED** [review]
I think this PR is very nice and it does more than the original purpose, "removing repeated content". Well done @amal-ghamdi .
I'll giving approval now and only leave some tiny thoughts in the comments.

File: chapter01/introduction_overview_of_cuqipy.md (hunk 1 above)
Comment (amal-ghamdi) [reply chapter01/introduction_overview_of_cuqipy.md:105]: Thank you @chaozg, this is a good point, I made a note of it in #68

**Comment by amal-ghamdi** [comment]
thank you @jakobsj and @chaozg for your review! I am merging. 

@chaozg, I took the liberty to resolve the two conversations from your comments since I tracked it elsewhere (in #68 ).

**Pull request merged**